server/src/main/java/org/elasticsearch/search/aggregations/metrics/PercentilesConfig.java

```diff
@@ -123,17 +124,29 @@ public static class TDigest extends PercentilesConfig {
         static final double DEFAULT_COMPRESSION = 100.0;
         private double compression;
 
+        private TDigestExecutionHint executionHint = TDigestExecutionHint.DEFAULT;
```

Comment:
Let me check if I understand correctly. Now default uses avl tree tdigest and high accuracy also defaults to this. And in a followup PR we will introduce a new enum value for best performance (which uses sorting and then falls back to merging tdigest impl) and default will basically do that too?

Reply:
Or, switch `TDigestState::Type::defaultValue()` to point to `HYBRID` instead of  `AVL_TREE`, single-liner.

Reply:
Right, default would point to hybrid (which uses both sorting and merging)